lms/djangoapps/gradebook/management/commands/tests/test_generate_gradebook_entries.py

```diff
@@ -29,7 +33,7 @@ class GenerateGradebookEntriesTests(TestCase):
     def setUp(self):
 
         # Turn off the signalling mechanism temporarily
-        settings._wrapped.default_settings.FEATURES['SIGNAL_ON_SCORE_CHANGED'] = False
+        #settings._wrapped.default_settings.FEATURES['SIGNAL_ON_SCORE_CHANGED'] = False
```

Comment:
Yeah that was the reason for one of the test in gradebook was failing. uncommenting it.